repose-aggregator/components/filters/valkyrie-authorization/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -61,18 +64,11 @@ class ValkyrieAuthorizationFilter @Inject()(configurationService: ConfigurationS
     configurationService.unsubscribeFrom(configurationFile, this)
   }
 
-  trait ValkyrieResult
-
-  case class UserInfo(tenantId: String, contactId: String) extends ValkyrieResult
-  case class UserPermissions(roles: Vector[String], devices: Vector[DeviceToPermission]) extends ValkyrieResult
-  case class DeviceToPermission(device: Int, permission: String)
-  case class ResponseResult(statusCode: Int, message: String = "") extends ValkyrieResult
-
   override def doFilter(servletRequest: ServletRequest, servletResponse: ServletResponse, filterChain: FilterChain): Unit = {
     val mutableHttpRequest = MutableHttpServletRequest.wrap(servletRequest.asInstanceOf[HttpServletRequest])
     val mutableHttpResponse = MutableHttpServletResponse.wrap(mutableHttpRequest, new HttpServletResponseWrapper(servletResponse.asInstanceOf[HttpServletResponse]))
 
-    def nullOrWhitespace(str: Option[String]): Option[String] = str.map { _.trim }.filter { !"".equals(_) }
+    def nullOrWhitespace(str: Option[String]): Option[String] = str.map(_.trim).filter(!"".equals(_))
```

Comment:
Nope, no functional differences in this example. Here, the IntelliJ formatter (at least the version I'm using with the default configuration) wants to spread the code blocks (everything between { and }) across multiple lines. Which makes sense, since that is the recommended style in the Scala style guide.

After trying to find a good explanation of the differences between the two, I came a much better explanation:
http://stackoverflow.com/a/4387118